### PR TITLE
New TTA page content - WIP

### PR DESCRIPTION
**Pull request by HettieS**
### Trello card
https://trello.com/c/G8Wig7Z8/1333-review-sarah-bhs-suggestion-for-a-tta-page-in-terms-of-web-content-suitability

### Context
add new TTA page with basic content

### CODE REVIEW

File: content/getting-a-teacher-training-adviser.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Their expertise can also help you overcome any particular challenges that may be getting in your way, such as caring responsibilities or working hours. 
+
+Your adviser will be just a call, text or email away.
+
+<p class="call-to-action__action">
+  <a href="/tta-service">Get a Teacher Training <span>Adviser</span></a>
+</p>
+
+Corinne wanted to become a chemistry teacher, and was paired with adviser Leilah
```
Comment (peteryates) [inline content/getting-a-teacher-training-adviser.md]:
```suggestion
Corinne wanted to become a chemistry teacher, and was paired with adviser Leilah.
```

**Comment by peteryates** [comment]
Looks like a good start! The document ending with three fairly long quotes back-to-back doesn't feel quite right to me though.

**Comment by HettieS** [comment]
cheers @peteryates, definitely agree the last section with the story quotes is a little weird. Can you think off top of your head any styling elements we could use on that section to improve the look?

**Comment by peteryates** [comment]
We do have a style for blockquotes that puts them in a yellow box, it's used quite heavily in the stories part of the site.

Screenshot from 2021-03-19 09-41-40

I wonder if picking a couple of relevant excerpts from the provided copy and using them within the document could work here?

Edit: To do a blockquote in markdown start the line with a `>`:

> Like this!

**Review by peteryates — DISMISSED** [review]
This looks fine to me 👍🏽

**Comment by peteryates** [comment]
Superseded by #364

**Pull request closed without merging**